docs/installation/presto-configuration.rst

```diff
@@ -9,74 +10,92 @@ you still may need to make some changes.
 
 
 Memory configuration
----------------------
+--------------------
 It is often necessary to change the default memory configuration based on your cluster's
 capacity. The default max memory for each Presto server is 16 GB, but if you have a lot of
 memory (say, 120GB/node), you may want to allocate more memory to Presto for better performance.
 
 In order to update the max memory value to 60 GB per node:
 
 1. Change the line in ``/etc/opt/prestoadmin/coordinator/jvm.config`` and
-``/etc/opt/prestoadmin/workers/jvm.config`` that says ``-Xmx16G`` to ``-Xmx60G``.
+   ``/etc/opt/prestoadmin/workers/jvm.config`` that says ``-Xmx16G`` to ``-Xmx60G``.
 
 2. Change the following lines in ``/etc/opt/prestoadmin/coordinator/config.properties``
-and ``/etc/opt/prestoadmin/workers/config.properties``: ::
+   and ``/etc/opt/prestoadmin/workers/config.properties``: ::
 
-    query.max-memory-per-node=8GB
-    query.max-memory=50GB
+        query.max-memory-per-node=8GB
+        query.max-memory=50GB
 
 
-to ::
+   to ::
 
-    query.max-memory-per-node=30GB
-    query.max-memory=<30GB * number of nodes>
+        query.max-memory-per-node=30GB
+        query.max-memory=<30GB * number of nodes>
```

Comment:
Shouldn't we refer to data skew? If there's a data skew, `max-memory-per-node` should be higher than `total_memory / number_of_nodes`.